toolchains/iOS.framework.cmake

```diff
@@ -3,7 +3,7 @@ include(${CMAKE_SOURCE_DIR}/toolchains/iOS.toolchain.cmake)
 message(STATUS "Build for iOS archs " ${CMAKE_OSX_ARCHITECTURES})
 
 set(FRAMEWORK_NAME TangramMap)
-set(FRAMEWORK_VERSION "1.0")
```

Comment:
Can we keep this consistent with major, minor and patch numbers here?

This will keep it consistent with what we have for android in https://github.com/tangrams/tangram-es/blob/master/android/tangram/gradle.properties#L2.

Also how are we planning on differentiating `SNAPSHOT`s with `RELEASE` versions?